build.gradle

```diff
@@ -400,6 +396,17 @@ uploadArchives {
                 ' for more information.' )
             throw new Exception("Missing sonatype username or password, please see the README for properties that must be specified for uploading an archive.")}
         )
-
     }
+
+distributions {
+    spark{} //added to allow an installSparkDist target which will also build the spark jar
+}
+
+installSparkDist.dependsOn downloadGsaLibFile, sparkJar
```

Comment:
Ablation, eh? Someone is on a phone :) Well, if we can't use `gradle install`, what do you guys think about my proposal of having `gradle installDist`, `gradle installSpark`, and `gradle installAll`?